packages/react/src/context/use-context-mutator.ts

```diff
@@ -39,6 +39,7 @@ export function useContextMutator(options: ContextMutationOptions = { defaultCon
     async (
       updatedContext: EvaluationContext | ((currentContext: EvaluationContext) => EvaluationContext),
     ): Promise<void> => {
+      // TODO: Needs to handle `openfeature` option like OpenFeatureProvider
```

Comment:
Once https://github.com/open-feature/js-sdk/pull/1317 is in, I suspect I can apply a similar approach to that here, where the client tracks a reference to the OpenFeatureAPI instance it came from, and we can use that here to infer the correct instance